docs/README.md

```diff
@@ -1,116 +1,145 @@
+You can use:
 
-<p align="center">
-  <img src="./images/providers-compute.png">
-</p>
+- Any Web IDE, such as
 
-Coder workspaces don't stop at compute. You can add storage buckets, secrets,
-sidecars and whatever else Terraform lets you dream up.
+  - [code-server](https://github.com/coder/code-server)
+  - [JetBrains Projector](https://github.com/JetBrains/projector-server)
+  - [Jupyter](https://jupyter.org/)
+  - And others
 
-[Learn more about templates.](./admin/templates/index.md)
+- Your existing remote development environment:
 
-## IDE Support
+  - [JetBrains Gateway](https://www.jetbrains.com/remote-development/gateway/)
+  - [VS Code Remote](https://code.visualstudio.com/docs/remote/ssh-tutorial)
+  - [Emacs](./user-guides/workspace-access/emacs-tramp.md)
 
-You can use any [Web IDE](./admin/templates/extending-templates/web-ides.md)
-([code-server](https://github.com/coder/code-server),
-[projector](https://github.com/JetBrains/projector-server),
-[Jupyter](https://jupyter.org), etc.),
-[JetBrains Gateway](https://www.jetbrains.com/remote-development/gateway/),
-[VS Code Remote](https://code.visualstudio.com/docs/remote/ssh-tutorial) or even
-a file sync such as [mutagen](https://mutagen.io/).
-
-<p align="center">
-  <img src="./images/ide-icons.svg" height=72>
-</p>
+- A file sync such as [Mutagen](https://mutagen.io/)
 
 ## Why remote development
 
-Migrating from local developer machines to workspaces hosted by cloud services
-is an
-[increasingly common solution for developers](https://blog.alexellis.io/the-internet-is-my-computer/)
-and
-[organizations alike](https://slack.engineering/development-environments-at-slack).
-There are several benefits, including:
+Remote development offers several benefits for users and administrators, including:
+
+- **Increased speed**
+
+  - Server-grade cloud hardware speeds up operations in software development, from
+    loading the IDE to compiling and building code, and running large workloads
+    such as those for monolith or microservice applications.
+
+- **Easier environment management**
+
+  - Built-in infrastructure tools such as Terraform, nix, Docker, Dev Containers, and others make it easier to onboard developers with consistent environments.
 
-- **Increased speed:** Server-grade compute speeds up operations in software
-  development, such as IDE loading, code compilation and building, and the
-  running of large workloads (such as those for monolith or microservice
-  applications)
+- **Increased security**
 
-- **Easier environment management:** Tools such as Terraform, nix, Docker,
-  devcontainers, and so on make developer onboarding and the troubleshooting of
-  development environments easier
+  - Centralize source code and other data onto private servers or cloud services instead of local developers' machines.
+  - Manage users and groups with [SSO](./admin/users/oidc-auth.md) and [Role-based access controlled (RBAC)](./admin/users/groups-roles#roles).
```

Comment:
Can you confirm the RBAC's location? Its a 404 now.

Reply:
seems to be working - [link](https://coder.com/docs/@ea-rewrite-readme/about#why-remote-development)

is it still 404ing for you?


https://github.com/user-attachments/assets/d4ada2ad-e4b7-4e8b-8a6d-5d650be24f3b